hiredis-client/lib/redis_client/hiredis_connection.rb

```diff
@@ -9,7 +9,23 @@ class HiredisConnection
     include ConnectionMixin
 
     class << self
+      def default_ca_file
+        OpenSSL::X509::DEFAULT_CERT_FILE
+      end
+
+      def default_ca_path
+        OpenSSL::X509::DEFAULT_CERT_DIR
+      end
```

Comment:
```suggestion
```

Let's not expose methods just to access constants.

Reply:
I initially added those method to ease testing, but I don't understand anything to minitest, mocking the thing was a bit complex so I dropped the idea.

Would you rather have me use the constant directly, or make those methods private ?